test/DurableTaskLifeCycleNotificationTest.cs

```diff
@@ -0,0 +1,361 @@
+﻿using System;
```

Comment:
Please add the copyright header to the top of this new file.

Reply:
We will add it. Thank you!